index.js

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line no-var
+var mockingcase;
```

Comment:
Is there a need to declare it here?!

Reply:
Just to fix an eslint error, but since I changed it to 'const', it's now unneeded.